src/gui/sharemanager.cpp

```diff
@@ -146,6 +146,12 @@ void Share::deleteShare()
     job->deleteShare(getId());
 }
 
+bool Share::isUserGroupShare(const ShareType type)
```

Comment:
that name is not telling me what it does
what are you checking here ?

Reply:
@mgallien is it better now?

Reply:
@mgallien Hmmm, but now, SonarCloud is somewhat unhappy 🤔

Reply:
it is better for me and I somehow like long names so I disagree with SonarCloud

Reply:
@mgallien The plan is to make a better name once we have separate implementation for circle shares. Overall, that `UserGroupShareWidget` is not doing more than just handling user and group share cases. Some refactoring is needed, but, this would require quite some changes.